### PR TITLE
8.0 product public category partner

### PR DESCRIPTION
**Pull request by foutoucour**
Simple module that add a relation between product.public.category and res.partner.

It was used to be able to make a domain to filter product.public.category by partner.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why many2one instead a many2many? Are you only enabling a product to one partner? I see also that functionally speaking, users expect the contrary: select which products a partner can see in the partner form, not going to product to select the partner.

**Comment by rafaelbn** [comment]
Please @foutoucour could you answer @pedrobaeza ? If not interested please close PR. Thanks

File: product_public_category_partner/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{
+    'name': 'Product Public Category Partner',
+    'version': '1.0',
+    'author': 'Savoir-faire Linux',
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'license': 'AGPL-3',
+    'category': 'Website',
+    'summary': 'Add partner_id field to product.public.category',
+    'description': """
```
Comment (hbrunn) [inline product_public_category_partner/__openerp__.py:32]: move this to `README.rst` and use the OCA template

File: product_public_category_partner/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Product Public Category Partner',
+    'version': '1.0',
+    'author': 'Savoir-faire Linux',
```
Comment (hbrunn) [inline product_public_category_partner/__openerp__.py:26]: add OCA

File: product_public_category_partner/models/product_public_category.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# #############################################################################
+
+import logging
+
+_logger = logging.getLogger(__name__)
```
Comment (hbrunn) [inline product_public_category_partner/models/product_public_category.py:25]: unused, remove

**Comment by rafaelbn** [comment]
Hi @hbrunn I'm sorry for you time in this PR, author doesn't touch it from 5 Dec 2014. I close.

**Pull request closed without merging**